Sources/SKLogging/SetGlobalLogFileHandler.swift

```diff
@@ -116,6 +116,10 @@ private func logToFile(message: String, logDirectory: URL, logFileMaxBytes: Int,
 private func setUpGlobalLogFileHandlerImpl(logFileDirectory: URL, logFileMaxBytes: Int, logRotateCount: Int) {
   logHandler = { @LogHandlerActor message in
     do {
+      // In addition to writing to the log file, also log to stderr, so LSP output is visible in the editor
+      // (e.g. VS Code's Output panel).
+      fputs(message + "\n", stderr)
+      
```

Comment:
Almost LGTM :). But now that we're printing this to stderr as well, I think we can remove the `\(message)` from the `fputs` below when the write to the log fails.

Reply:
Fixed, thanks :)